plugins/TodayAtMun/__init__.py

```diff
@@ -0,0 +1,57 @@
+import discord
+from discord.ext import commands
+import typing
+
+from Bot import bot
+from Plugin import AutomataPlugin
+from plugins.TodayAtMun.DiaryParser import DiaryParser
+from plugins.TodayAtMun.Today import Today
+
+
+class TodayAtMun(AutomataPlugin):
+    def __init__(self, manifest, bot: commands.Bot):
+        super().__init__(manifest, bot)
+        self.parse = DiaryParser()
+        self.tod = Today(self.parse.diary)
```

Comment:
Would be nice to get a more descriptive variable name than `tod`

Reply:
yes for sure